abi/bigUIntValue_test.go

```diff
@@ -0,0 +1,48 @@
+package abi
```

Comment:
Existing file was split into two. Additionally, `BigUIntValue` must be passed by reference.